front/src/applications/stdcm/views/StdcmView.tsx

```diff
@@ -109,17 +111,17 @@ const StdcmView = () => {
      */
     const lastSimulation = simulationsList[simulationsList.length - 1];
     const isSimulationAlreadyListed = isEqual(lastSimulation?.inputs, currentSimulationInputs);
-    const isSimulationOutputsComplete =
-      stdcmResults?.stdcmResponse && stdcmResults?.speedSpaceChartData?.formattedPathProperties;
+    const isSimulationOutputsComplete = stdcmResults?.stdcmResponse || hasConflicts;
 
-    if (isSimulationOutputsComplete || isStdcmResultsEmpty) {
+    if (isSimulationOutputsComplete) {
       const newSimulation = {
         ...(isSimulationAlreadyListed
           ? { ...lastSimulation }
           : {
               id: simulationsList.length + 1,
               creationDate: new Date(),
               inputs: currentSimulationInputs,
+              conflicts: stdcmTrainConflicts,
```

Comment:
Conflicts aren't an input but an output. We should adapt the output to accept `conflicts` and `pathProperties`.

Suggestion : 
```
export type StdcmSimulationOutputs = {
  pathProperties: ManageTrainSchedulePathProperties;
} & (
  | {
      results: StdcmSuccessResponse;
      speedSpaceChartData: SpeedSpaceChartData;
    }
  | { conflicts: Conflict[] }
);
```